core/pkg/hostsensorutils/hostsensorgetfrompod.go

```diff
@@ -269,6 +275,16 @@ func (hsh *HostSensorHandler) CollectResources() ([]hostsensor.HostSensorDataEnv
 		res = append(res, kcData...)
 	}
 
+	// GetControlPlaneInfo
+	kcData, err = hsh.GetControlPlaneInfo()
+	if err != nil {
+		addInfoToMap(KubeProxyInfo, infoMap, err)
```

Comment:
KubeProxyInfo --> ControlPlaneInfo

Reply:
Resolved with force push. It won't show here